spec/features/project_spec.rb

```diff
@@ -18,5 +18,15 @@
       expect(page).to have_content('Book reviews in space')
     end
 
+    scenario 'unsuccessful update' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.